volume/local/local.go

```diff
@@ -188,7 +187,8 @@ func (r *Root) Get(name string) (volume.Volume, error) {
 
 func (r *Root) validateName(name string) error {
 	if !volumeNameRegex.MatchString(name) {
-		return derr.ErrorCodeVolumeName.WithArgs(name, utils.RestrictedNameChars)
+		err := fmt.Errorf("%q includes invalid characters for a local volume name, only %q are allowed", name, utils.RestrictedNameChars)
+		return errors.NewBadRequestError(err)
```

Comment:
you're probably the one that made that `ErrorCodeVolumeName` to return a bad request status :stuck_out_tongue: